src/app/portlets/content-types/fields/service/field-drag-drop.service.ts

```diff
@@ -31,6 +33,12 @@ export class FieldDragDropService {
             } else if (source === 'target') {
                 this._fieldDropFromTarget.next();
             }
+        } else if (dragType === 'fields-row-bag') {
```

Comment:
This whole method it's doing too much things, you should break this into method for fields and row.